package.json

```diff
@@ -253,6 +253,39 @@
         ],
         "visible": true
       },
+      {
+        "permissionName": "ui-users.settings.customfields.view",
+        "displayName": "Settings (Users): Can view custom fields",
+        "subPermissions": [
+          "customfields.collection.get",
+          "customfields.item.get",
```

Comment:
It should be `custom.fields.collection.get` and `custom.fields.item.get`
Please, correct the names of other backend permissions too
Also, `configuration.entries.collection.get` should be added to the subpermissions too